QualificationChecker/Controllers/QualificationClass.cs

```diff
@@ -0,0 +1,131 @@
+﻿using QualificationChecker.Models;
+
+namespace QualificationChecker.Controllers
+{
+    public abstract class QualificationManager
```

Comment:
Is there a need to make an abstract class? (Only seeing one implementation)

Reply:
Not sure wanted to leave the door open for it but is not required for it now.